next.config.cjs

```diff
@@ -0,0 +1,4 @@
+﻿const withSass = import('@zeit/next-sass');
```

Comment:
I think you can undo this change - we fixed this by removing `type: module` from package.json